src/screens/DiscoveryScreenV2/elements/RecentSearch.js

```diff
@@ -104,7 +104,7 @@ const styles = StyleSheet.create({
   recentContainer: {
     paddingTop: 18,
     paddingBottom: 15,
-    backgroundColor: COLORS.white
+    backgroundColor: COLORS.almostBlack
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.white` to `COLORS.almostBlack` for the `backgroundColor` property of `recentContainer` might significantly alter the appearance of this component. Ensure that this color change aligns with the overall design and accessibility guidelines, particularly in terms of contrast ratios for text readability.
```diff
-    backgroundColor: COLORS.white
+    backgroundColor: COLORS.almostBlack
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->